medical_center/__manifest__.py

```diff
@@ -0,0 +1,27 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016 LasLabs Inc.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+{
+    "name": "Medical Centers",
+    "summary": "Adds a concept of Medical Centers to Patients.",
+    "version": "10.0.1.0.0",
+    "category": "Medical",
+    "website": "https://laslabs.com",
+    "author": "LasLabs, Odoo Community Association (OCA)",
+    "license": "AGPL-3",
+    "application": False,
+    "installable": True,
+    "depends": [
+        "medical",
+    ],
+    "data": [
+        "views/medical_center.xml",
+        "views/medical_patient.xml",
+        "views/medical_menu.xml",
```

Comment:
It does not follow the contribution guidlines: https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md#file-naming


But I wonder if contribution guidelines should not be updated... something more like: 

- addons/<my_module_name>/
  - menu
    - `For a menu: <root_menu_name>_menu.xml`